docs/xks/operator-guide/cve.md

```diff
@@ -0,0 +1,37 @@
+---
+id: cve
+title: CVE Information
+---
+
+This page is a way for us to inform about CVE:s that we see in the wilde and have given extra attention to.
+
+This list is not complete but we try to keep it up to date around CVE:s related to our space wether they are Kubernetes, Linux och developer centric.
```

Comment:
"wether" -> "whether"
"och" -> "and"

However, I think it would be better to just say "... that are relevant to XKS and its usage." or some such.